sourcetracker/_plot.py

```diff
@@ -10,16 +10,306 @@
 
 import seaborn as sns
 import matplotlib.pyplot as plt
+import os
 
 
-def plot_heatmap(mpm, cm=plt.cm.viridis, xlabel='Sources', ylabel='Sinks',
-                 title='Mixing Proportions (as Fraction)'):
-    '''Make a basic mixing proportion histogram.'''
-    fig = plt.figure()
-    ax = fig.add_subplot(1, 1, 1)
-    sns.heatmap(mpm, vmin=0, vmax=1.0, cmap=cm, annot=True, linewidths=.5,
-                ax=ax)
-    ax.set_xlabel(xlabel)
-    ax.set_ylabel(ylabel)
-    ax.set_title(title)
-    return fig, ax
+class ST_graphs:
+    def __init__(self, mpm, output_dir,
+                 title='Mixing Proportions', color='viridis'):
+        self.file = output_dir
+        self.mpm = mpm.T
+        self.title = title
+        self.color = color
+        self.out_name = title.replace(" ", "_")
+        """
+        Parameters
+        ----------
+        self.file string
+            output directory name given from input
+        self.mpm dataframe
+            mixing proportion result from gibbs
+        self.title string
+            title
+        self.color string
+            color scheme for heatmaps pulled from matplotlib
+        self.out_name string
+            output name spaces are replaced with _
+        """
+    def ST_heatmap(self, unknowns=True, annot=True,
+                   xlabel='Sources', ylabel='Sinks', vmax=1.0):
+        """
+        Default Heatmap default is true
+        Standard heat map altered for custom
+        shape and direct png save function
```

Comment:
```suggestion
        Default Plot for Gibbs method.
```
be sure to only include comments that make sense solely in the context of the code, not ones that need the context of the PR.  (I.e saying what you altered in the code in a specific way only makes sense in the context of the PR)